graphql-dgs-spring-boot-micrometer/src/main/kotlin/com/netflix/graphql/dgs/metrics/micrometer/dataloader/BatchLoaderInterceptor.kt

```diff
@@ -24,12 +23,8 @@ internal class BatchLoaderInterceptor(
                 logger.debug("Finished timer[{}] for BatchLoader.", ID)
                 timerSampler.stop(
                     Timer.builder(ID)
-                        .tags(
-                            listOf(
-                                Tag.of(GqlTag.LOADER_NAME.key, name),
-                                Tag.of(GqlTag.LOADER_BATCH_SIZE.key, result.size.toString())
-                            )
-                        ).register(registry)
+                        .tags(GqlTag.LOADER_BATCH_SIZE.key, result.size.toString(), GqlTag.LOADER_NAME.key, name)
```

Comment:
Is this changing how the tags are added? With your update above, do they all just get added individually as 4 tags instead of 2?

Reply:
It's still adding the 2 tags; this method takes a varargs String array and expects an even number of args representing the key / value pairs. Calling this instead avoids needing to create a List; it's going to create a Tags instance under the hood either way. . The tag keys are re-ordered to avoid needing to actually do any sorting as a sort of micro-optimization.